src/components/Comments.jsx

```diff
@@ -27,13 +27,21 @@ const Comments = ({ article }) => {
     <div>
       <section>
         <ul>
-          {comments.map((comment, index) => {
-            return (
-              <li key={index}>
-                <CommentCard comment={comment} />
-              </li>
-            );
-          })}
+          {comments
+            .sort((a, b) =>
+              a.created_at > b.created_at
+                ? -1
+                : a.created_at < b.created_at
+                ? 1
+                : 0
+            )
+            .map((comment, index) => {
```

Comment:
does your backend support a sort_by query?

It's not ideal to do data sorting in the front end

Reply:
Unfortunately sort_by is not supported atm when fetching the comments on that endpoint.
It should be added, so there would not be sorting needed.